src/v2/guide/typescript.md

```diff
@@ -0,0 +1,98 @@
+---
+title: TypeScript Support
+type: guide
+order: 25
+---
+
+## Official Declaration Files
+
+A strong type system can help prevent many potential runtime errors, especially as applications grow. That's why Vue ships with [official type declarations](https://github.com/vuejs/vue/tree/dev/types) for [TypeScript](https://www.typescriptlang.org/) - not only in Vue core, but also [for Vue Router](https://github.com/vuejs/vue-router/tree/dev/types) and [for Vuex](https://github.com/vuejs/vuex/tree/dev/types) as well.
```

Comment:
Would you mind replace `strong` with `static`? TypeScript isn't type sound in language design. `static` is probably a better choice to express compile time checking.